sessionctx/variable/tidb_vars.go

```diff
@@ -214,6 +214,9 @@ const (
 
 	// TiDBReadStaleness indicates the staleness duration for following statement
 	TiDBReadStaleness = "tidb_read_staleness"
+
+	// TiDBEnablePaging indicates whether paging is enabled in coprocessor requests.
+	TiDBEnablePaging = "tidb_enable_paging"
```

Comment:
May need to add a `TODO` here that system variable should be removed in release version.

Reply:
Do you mean when paging is stable, we should hide this variable and every request will go through paging by default?